src/foam/nanos/notification/email/EmailServiceDAO.js

```diff
@@ -9,14 +9,27 @@ foam.CLASS({
   name: 'EmailServiceDAO',
   extends: 'foam.dao.ProxyDAO',
 
+  requires: [
+    'foam.nanos.notification.email.EmailMessage',
+    'foam.nanos.notification.email.EmailService'
+  ],
+
+  imports: [
+    'email?'
+  ],
+
   properties: [
     {
       name: 'emailService',
       documentation: `This property determines how to process the email.`,
       of: 'foam.nanos.notification.email.EmailService',
       class: 'FObjectProperty',
-      javaGetter:
-      `return (foam.nanos.notification.email.EmailService)getX().get("email");`
+      factory: function() {
+        return this.email;
```

Comment:
Unless you plan on implementing the JS put_ method, I wouldn't include this factory.